src/test/java/org/folio/spring/controller/TenantControllerIT.java

```diff
@@ -42,6 +45,21 @@ DataSource dataSource() {
     }
   }
 
+  @Test
+  void canCallManagementEndpointWithoutTenantHeader(
+      @Value("${management.endpoints.web.base-path:/admin}") String mgmtBasePath) throws Exception {
+    mockMvc.perform(get(mgmtBasePath + "/health"))
+      .andExpect(status().is(404)) // will be 200 if actuator is added
+      .andExpect(content().string(not(startsWith("x-okapi-tenant header must be provided"))));
```

Comment:
Consider refactor the message to a constant and reference it from different places

Reply:
sure